src/Uno.UWP/Storage/Helpers/StorageFile.wasm.cs

```diff
@@ -13,11 +13,14 @@ namespace Windows.Storage.Helpers;
 
 partial class StorageFileHelper
 {
+	private static string assets { get; set; } = string.Empty;
 	private static async Task<bool> FileExistsInPackage(string fileName)
 	{
-		var assetsUri = AssetsPathBuilder.BuildAssetUri("uno-assets.txt");
-
-		var assets = await WebAssemblyRuntime.InvokeAsync($"Windows.Storage.AssetManager.DownloadAssetsManifest(\'{assetsUri}\')");
+		if (string.IsNullOrEmpty(assets))
+		{
+			var assetsUri = AssetsPathBuilder.BuildAssetUri("uno-assets.txt");
+			assets = await WebAssemblyRuntime.InvokeAsync($"Windows.Storage.AssetManager.DownloadAssetsManifest(\'{assetsUri}\')");
```

Comment:
I don't understand this change, what is it needed for?

Reply:
Actually, I looked at the latest change only. You should be able to use this method instead:

https://github.com/unoplatform/uno/blob/5c6bd5fcd9ad846448db6c0f6029d5a331630c46/src/Uno.UWP/Storage/Helpers/AssetsManager.wasm.cs#L30